src/filters/FilterNumbers.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import Radium from 'radium';
 import {connect} from 'react-redux';
+import _ from "lodash";
```

Comment:
If the idea is to use just what we need from lodash this is importing the whole lib